test_check_dvcs.py

```diff
@@ -20,8 +20,8 @@ class TestDoesStringStartWithJira:
             ('other stuff AAP-4444 jira in the middle', None),
         ],
     )
-    def test_does_string_start_with_jira_function(self, input, expected_return):
-        result = check_dvcs.does_string_start_with_jira(input)
+    def test_does_string_contain_jira_function(self, input, expected_return):
+        result = check_dvcs.does_string_contain_jira(input)
         assert result == expected_return
```

Comment:
This is a follow up to my previous comment, shouldn't the expected_return be a match instead of None